src/identity.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+namespace Lambdish\Phunctional;
+
+/**
+ * It wraps a value into a Closure, which return the same value that is passed as argument whenever is called
+ *
+ * @since v1.0.7
+ *
+ * @param mixed $value any type of value
+ *
+ * @return \Closure
+ */
+function identity($value)
+{
+    return function () use ($value) {
+        return $value;
+    };
+}
```

Comment:
```php
function identity($value)
{
    return $value;
}
```

? :)

identity = `f(x) = x`
Here we have: `f(x) = f() -> x`

Reply:
constant = f() = c